pyproject.toml

```diff
@@ -85,7 +85,7 @@ ninja.version = ">=1.10"
 build-dir = "build/{wheel_tag}/{build_type}"
 
 # Only build the Python bindings target
-build.targets = ["ir"]
+build.targets = ["ir", "na"]
```

Comment:
Would it make sense to add an interface target in the `src/python/CMakeLists.txt` file after including all the subdirectories that links all the bindings targets together.
That would let us use a single string here.
Maybe its also fine to leave this as is, given how this will change with #662 again anyway.